tests/e2e/cucumber/features/shares/share.feature

```diff
@@ -22,10 +22,10 @@ Feature: share
       | lorem.txt     | folder_to_shared       |
       | lorem-big.txt | folder_to_customShared |
     When "Alice" shares the following resource using the sidebar panel
-      | resource               | recipient | type | role                                  | resourceType |
-      | folder_to_shared       | Brian     | user | Can edit                              | folder       |
-      | shared_folder          | Brian     | user | Can edit                              | folder       |
-      | folder_to_customShared | Brian     | user | custom_permissions:read,create,delete | folder       |
+      | resource               | recipient | type | role       | resourceType |
+      | folder_to_shared       | Brian     | user | Can edit   | folder       |
+      | shared_folder          | Brian     | user | Can edit   | folder       |
+      | folder_to_customShared | Brian     | user | Can edit   | folder       |
```

Comment:
Mh does this still have any value now? Maybe it should be removed (in follow up)

Reply:
https://github.com/owncloud/web/pull/10677